tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -352,10 +352,22 @@ public function theUserChangeTheExpirationOfThePublicLinkNamedForTo($linkName, $
 		$session = $this->getSession();
 		$this->theUserOpensTheShareDialogForFileFolder($name);
 		$this->theUserHasOpenedThePublicLinkShareTab();
-		$this->publicShareTab->editLink($session, $linkName, null, null, null, $date);
+		$this->publicSharingPopup = $this->publicShareTab->editLink($session, $linkName, null, null, null, $date);
 		$this->publicShareTab->waitForAjaxCallsToStartAndFinish($session);
 	}
 
+	/**
+	 * @Then the user should see an error message on the public link popup saying :message
+	 *
+	 * @param string $message
+	 *
+	 * @return void
+	 */
+	public function theUserShouldSeeAnErrorMessageOnThPublicLinkPopupSaying($message) {
+		$errormessage = $this->publicSharingPopup->getErrorMessage();
```

Comment:
Instead of saving to `publicSharingPopup` above, would this work? This way, we will always have fresh `publicSharingPopup` (I could be wrong here).
```suggestion
		$publicSharingPopup = $this->publicShareTab->updateSharingPopup();
		$errormessage = $publicSharingPopup->getErrorMessage();
```

P.S. Currently, `updateSharingPopup` doesnot return anything.

Reply:
As per the discussion with @paurakhsharma, the both suggestions don't make much difference.